### PR TITLE
chore(version): Bump version

### PR DESCRIPTION
**Pull request by dnys1**
### Breaking Changes
- chore(auth)!: Chain stack traces in state machine
- chore(core)!: Chain stack traces for state machines
- feat(api)!: custom primary key support for GraphQL model helpers ([#2606](https://github.com/aws-amplify/amplify-flutter/pull/2606))

### Fixes
- fix(api): early call of Amplify creates wrong instance of AmplifyClass
- fix(api): write null values in ModelMutations.create() unless owner field ([#2679](https://github.com/aws-amplify/amplify-flutter/pull/2679))
- fix(auth): Await `signInUri` in Hosted UI platform ([#2706](https://github.com/aws-amplify/amplify-flutter/pull/2706))
- fix(auth): Transform session expired exceptions ([#2688](https://github.com/aws-amplify/amplify-flutter/pull/2688))
- fix(storage): GetUrl signing

### Features
- feat(api): GraphQL Subscription Where Filter ([#2650](https://github.com/aws-amplify/amplify-flutter/pull/2650))
- feat(aws_common): add openRead API for AWSFile
- feat(storage): optimize part size for multipart upload
- feat(storage): web implementation of transfer database using local storage ([#2631](https://github.com/aws-amplify/amplify-flutter/pull/2631))

Updated-Components: aws_common, Amplify Flutter, Amplify Dart, Amplify UI, DB Common

### CODE REVIEW

File: packages/amplify_core/lib/src/amplify_class.dart [hunk 1]
```diff
@@ -133,7 +133,7 @@ abstract class AmplifyClass {
   static AmplifyClass? instance;
 
   /// The library version.
-  String get version => '1.0.0-next.2';
+  String get version => '1.0.0-next.6';
```
Comment (ragingsquirrel3) [inline packages/amplify_core/lib/src/amplify_class.dart:136]: Thanks for changing this, but why was this so far behind? Does it have to be changed manually?

Comment (dnys1) [reply packages/amplify_core/lib/src/amplify_class.dart:136]: Yeah opened a separate PR to automate this

File: packages/api/amplify_api/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+## 1.0.0-next.6
+
+- Minor bug fixes and improvements
```
Comment (ragingsquirrel3) [inline packages/api/amplify_api/CHANGELOG.md:3]: request: Could this have relevant changes?

```
### Breaking Changes
* feat(api)!: custom primary key support for GraphQL model helpers ([#2606](https://github.com/aws-amplify/amplify-flutter/pull/2606))
Fixes

### Fixes
* fix(api): early call of Amplify creates wrong instance of AmplifyClass
* fix(api): write null values in ModelMutations.create() unless owner field ([#2679](https://github.com/aws-amplify/amplify-flutter/pull/2679))

### Features
* feat(api): GraphQL Subscription Where Filter ([#2650](https://github.com/aws-amplify/amplify-flutter/pull/2650))
```

File: packages/storage/amplify_storage_s3/CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,7 @@
+## 1.0.0-next.6
+
+- Minor bug fixes and improvements
+
```
Comment (ragingsquirrel3) [inline packages/storage/amplify_storage_s3/CHANGELOG.md:4]: question: Should any of the changes from dart package go here as well?

**Review by ragingsquirrel3 — APPROVED** [review]
approved w a few  non-blocking changelog nits and questions

File: packages/api/amplify_api/CHANGELOG.md (hunk 2 above)
Comment (dnys1) [reply packages/api/amplify_api/CHANGELOG.md]: Done 👍

File: packages/storage/amplify_storage_s3/CHANGELOG.md (hunk 3 above)
Comment (dnys1) [reply packages/storage/amplify_storage_s3/CHANGELOG.md:9]: Done 👍

**Review by Jordan-Nelson — APPROVED** [review]

**Pull request merged**